ansible/roles/test/tasks/arp_clear.yml

```diff
@@ -0,0 +1,151 @@
+######################################################################
+## This playbook tests that we are able to clear ARP entries using the
+## sonic-clear utility.
+######################################################################
+
+- name: Define some common test values
```

Comment:
Is there any way to verify these values are not used by some random chance in the device?